src/screens/login/Login.tsx

```diff
@@ -1,115 +1,44 @@
 import { useContext, useEffect, useState } from 'react';
-import { View, Text, Image, TouchableHighlight, Linking } from 'react-native';
+import { View, Text, Image, Linking, TouchableOpacity } from 'react-native';
 import Toast from 'react-native-toast-message';
 
 import { Tuneify } from '@assets/images';
 import GradientScreen from '@components/gradientScreen';
 import Icon from '@components/icon';
 import LoadingView from '@components/loadingView';
-import TokenContext from '@config/TokenContext';
-import { IconFamily, isIos } from '@constants';
+import { IconFamily, REQUEST_STATUS, STORAGE_KEY } from '@constants';
 import { _post, _postAccount } from '@network/instanceMethods';
-import ApiConstants from '@network/apiConstants';
-import { useAppSelector } from '@store';
-import { Colors, GlobalThemedStyles } from '@themes';
-import {
-  appendSearchParams,
-  colorWithOpacity,
-  generateRandomString,
-  parseUrl,
-} from '@utility/helpers';
+import { useAppDispatch, useAppSelector } from '@store';
+import { authorizeUser, requestAccessTokenViaCode } from '@store/reducers/auth';
+import { GlobalThemedStyles } from '@themes';
+import { parseUrl } from '@utility/helpers';
+import StorageManager from '@utility/storage';
 
 import { displayName as appName } from '../../../app.json';
 import ThemedStyles from './styles';
 
 const Login = () => {
-  const isDark = useAppSelector(({ theme }) => theme.isDark);
-  const { saveAccessToken, saveRefreshToken, login } = useContext(TokenContext);
-
-  const [loading, setLoading] = useState(false);
-  const [loadingProcessInfo, setLoadingProcessInfo] = useState('');
+  const { loading, loadingProcess, error } = useAppSelector(({ user }) => user);
```

Comment:
comment the difference b/w "loading" and "loadingProcess"

Reply:
loading process provides info text, for what process loading is happening